src/core/connection.c

```diff
@@ -3840,7 +4082,20 @@ QuicConnRecvFrames(
             } else if (Status == QUIC_STATUS_OUT_OF_MEMORY) {
                 return FALSE;
             } else {
-                if (Status != QUIC_STATUS_INVALID_STATE) {
+                if (Status == QUIC_STATUS_VER_NEG_ERROR) {
+                    if (QuicBindingQueueStatelessOperation(
+                            Connection->Paths[0].Binding,
+                            QUIC_OPER_TYPE_VERSION_NEGOTIATION,
+                            CxPlatDataPathRecvPacketToRecvData(Packet))) {
+                        //
+                        // Packets that elicit a VN response don't get counted as received
+                        // if they're deferred. Update that count here to ensure it gets counted.
+                        //
+                        Connection->Stats.Recv.TotalPackets += 2;
+                        Packet->ReleaseDeferred = TRUE;
```

Comment:
We don't need to do this. For one, we don't even know if the packet was completely valid...

Reply:
Ok, how about we just increment by 1 so when the `TotalPackets` counter is decremented (because an error was returned), it doesn't underflow?